src/data/dialogmotekandidat/types/dialogmoteikkeaktuellTypes.ts

```diff
@@ -0,0 +1,11 @@
+export enum IkkeAktuellArsak {
+  ARBEIDSTAKER_AAP = "ARBEIDSTAKER_AAP",
+  ARBEIDSTAKER_DOD = "ARBEIDSTAKER_DOD",
+  DIALOGMOTE_AVHOLDT = "DIALOGMOTE_AVHOLDT",
+}
```

Comment:
Vet vi at disse er uttømmende for hvilke årsaker det kan være? 😊 Eller bør vi ha en `OTHER`?

Reply:
Tviler på at det er ønskelig med en ANNEN-sekkepost her - da bør de heller sette unntak vil jeg tro.

Reply:
Så lenge Stine har tatt stilling til det er vi vel good foreløpig 😊 Bare opptatt av at vi ikke antar noe om bruk som kan ende opp med å være feil.